pkg/metrics/collector/pod_ip_metrics.go

```diff
@@ -224,5 +290,9 @@ func (c *podIpMetricsCollector) Update(ch chan<- prometheus.Metric) error {
 	ch <- prometheus.MustNewConstMetric(dualStackErrorCountDesc, prometheus.GaugeValue, float64(c.dualStackErrorCount))
 	ch <- prometheus.MustNewConstMetric(duplicateIpCountDesc, prometheus.GaugeValue, float64(c.duplicateIpCount))
 
+	ch <- prometheus.MustNewConstMetric(ipReusedMinDesc, prometheus.GaugeValue, float64(c.reusedIps.min))
+	ch <- prometheus.MustNewConstMetric(ipReusedAvgDesc, prometheus.GaugeValue, float64((c.reusedIps.sum / float64(c.reusedIps.count))))
+	ch <- prometheus.MustNewConstHistogram(ipReusedHistogramDesc, c.reusedIps.count, c.reusedIps.sum, c.reusedIps.buckets)
```

Comment:
Is it possible to get the min and average from a histogram metric? If so, the other metrics may not be required.

Reply:
I didn't find a way to get that from histogram, so I keep the original metrics.